jans-cedarling/cedarling/src/authz/entities/create.rs

```diff
@@ -164,9 +171,9 @@ fn entity_meta_attributes(
 fn build_entity_attributes(
     schema: &CedarSchemaJson,
     parsed_typename: &EntityParsedTypeName,
-    data: &TokenPayload,
+    tkn_data: &TokenData,
```

Comment:
Why we need prefix `tkn_` if type clearly indicates what it is.

Reply:
because the `schema`, `parsed_typename` are also technically *data* it's better to be specific than let others assume they know what you're thinking.